erpnext_quota/erpnext_quota/doctype/quota_document_limit_detail/quota_document_limit_detail.json

```diff
@@ -0,0 +1,49 @@
+{
+ "actions": [],
+ "allow_rename": 1,
+ "creation": "2023-03-01 21:01:37.958915",
+ "doctype": "DocType",
+ "editable_grid": 1,
+ "engine": "InnoDB",
+ "field_order": [
+  "document_type",
+  "limit",
+  "period"
```

Comment:
Add usage too and update on each validation

Reply:
I took a different approach to reduce the calls, instead the usage info updates the usage when opened.